File: Java/LeetCode/Find Words Containing Character.java
// https://leetcode.com/problems/find-words-containing-character

class Solution {
  
    public List<Integer> findWordsContaining(String[] words, char x) {
        List<Integer> indices = new ArrayList<>();

        for (int i = 0; i < words.length; ++i) {
            if (words[i].contains("" + x)) {
                indices.add(i);
            }
        }

        return indices;
    }
  
}

File: Kotlin/LeetCode/Find Words Containing Character.kt
// https://leetcode.com/problems/find-words-containing-character/

class Solution {
  
    fun findWordsContaining(words: Array<String>, x: Char): List<Int> {
        var indices = arrayListOf<Int>();

        for (i in 0 until words.size) {
            if (words[i].contains("" + x)) {
                indices.add(i);
            }
        }

        return indices;
    }
    
}
